DuckDuckGoVPN/DuckDuckGoVPNAppDelegate.swift

```diff
@@ -375,6 +375,10 @@ final class DuckDuckGoVPNAppDelegate: NSObject, NSApplicationDelegate {
         // Load cached config (if any)
         privacyConfigurationManager.reload(etag: configurationStore.loadEtag(for: .privacyConfiguration), data: configurationStore.loadData(for: .privacyConfiguration))
 
+        // It's important for this to be set-up after the privacy configuration is loaded
+        // as it relies on it for the remote feature flag.
+        TipKitAppEventHandler(featureFlagger: featureFlagger).appDidFinishLaunching()
```

Comment:
Even though we're not showing tips in the VPN menu app, I left this in, in hopes we'll be able to eventually enable it again.